### PR TITLE
Add MintHCM to HR

### PR DESCRIPTION
**Pull request by phlbnks**
Adding MintHCM to HR

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Miscellaneous`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] If you are adding software forked from another project, please provide/link to a clear list of differences between both.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1103,6 +1103,7 @@ A [human resources management system](https://en.wikipedia.org/wiki/Human_resour
 
 - [admidio](https://www.admidio.org/) - Admidio is a free open source user management system for websites of organizations and groups. The system has a flexible role model so that it’s possible to reflect the structure and permissions of your organization. ([Demo](https://www.admidio.org/demo/), [Source Code](https://github.com/Admidio/admidio)) `GPL-2.0` `PHP`
 - [IceHrm](https://icehrm.com/) - IceHrm employee management system allows companies to centralize confidential employee information. ([Demo](https://icehrm.com/demo.php), [Source Code](https://github.com/gamonoid/icehrm)) `Apache-2.0` `PHP`
+- [MintHCM](https://minthcm.org/) - A professional tool for Human Capital Management. It's based on two popular, well-known business applications: SugarCRM Community Edition and SuiteCRM. ([Demo](https://minthcm.org/demo/) - requires sign-up, [Source Code](https://github.com/minthcm/minthcm)) `GPL-3.0` `PHP`
```
Comment (KieranRobson) [inline README.md:1106]:
```suggestion
- [MintHCM](https://minthcm.org/) - A professional tool for Human Capital Management. It's based on two popular, well-known business applications: SugarCRM Community Edition and SuiteCRM. ([Demo](https://minthcm.org/demo/), [Source Code](https://github.com/minthcm/minthcm)) `AGPL-3.0` `PHP`
```

Couple of changes to be made

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Software older than 4 months ✅

Couple changes to be made!

**Comment by nodiscc** [comment]
- Is there a reason why [commits](https://github.com/minthcm/minthcm/commits/master) are huge changesets without any description of what they do? (e.g. https://github.com/minthcm/minthcm/commit/0faf4ae36ea974cce4a437380c99c22d264e7b16)
- The description says it is partly based on "SugarCRM Community Edition" but [Wikipedia](https://en.wikipedia.org/wiki/SugarCRM#License) states that SugarCRM has been entirely proprietary since 2014. Are you sure the project contains no proprietary source code?

**Comment by phlbnks** [comment]
@nodiscc You would have to ask the maintainers - I have no affiliation with the project other than having discovered it recently! I can only take them at their work regarding the code - e.g. they say "open code of both SugarCRM CE and SuiteCRM we were able to reintroduce some features, redesign other, and provide brand new system to the users" which suggests to me they built off the last available OS code from those Projects.

**Review by 7kom95 — APPROVED** [review]

**Comment by nodiscc** [comment]
> You would have to ask the maintainers - I have no affiliation with the project other than having discovered it recently!

That makes sense. Did you try the software for a while before submitting it here? Is it working as expected?

**Comment by nodiscc** [comment]
Hi,
thank you for your contribution. Today an improved version of awesome-selfhosted goes live at https://awesome-selfhosted.net/. Please check the [release announcement](https://github.com/awesome-selfhosted/awesome-selfhosted/releases/tag/1.0.0) for more details.

Contributions should now be made through https://github.com/awesome-selfhosted/awesome-selfhosted-data. If you are still interested in having this software added to the list, please check contribution guidelines there, and open a new PR/issue.

**Pull request closed without merging**